test/test-gcp-auth.c

```diff
@@ -311,4 +352,6 @@ _mongocrypt_tester_install_gcp_auth (_mongocrypt_tester_t *tester)
    INSTALL_TEST (_test_encrypt_with_credentials);
    INSTALL_TEST (_test_createdatakey_with_accesstoken);
    INSTALL_TEST (_test_encrypt_with_accesstoken);
+   INSTALL_TEST (_test_createdatakey_with_wrong_kms_provider_configured);
+   INSTALL_TEST (_test_createdatakey_with_wrong_kms_provider_empty);
```

Comment:
Though I originally pointed to this file, suggest moving these two tests to `test-mongocrypt-ctx-setopt.c`. The behavior change is option validation and not specific to `gcp`.